redwood-protocol-host/src/commonMain/kotlin/app/cash/redwood/protocol/host/ProtocolBridge.kt

```diff
@@ -141,8 +141,11 @@ public class ProtocolBridge<W : Any>(
     // Apply deferred removes. Do this last so properties + modifiers always find their widget,
     // even if it's marked for death.
     for (removedId in removedIds) {
```

Comment:
I think this was an artifact of when you were sorting changes. If properties or modifiers ever came in after a remove an exception would be thrown because the map lookup by ID would fail.

I have removed this in #1920

Reply:
Awesome.